test/func_cpu_trap_flag.py

```diff
@@ -0,0 +1,88 @@
+
+from common_framework import mkfile, mkcom
+from cpuinfo import get_cpu_info
+
+
+def cpu_trap_flag(self, cpu_vm):
+
+    config = """\
+$_hdimage = "dXXXXs/c:hdtype1 +1"
+$_floppy_a = ""
+$_cpu_emu = "off"
+$_cpu_vm = "%s"
+""" % cpu_vm
+
+    mkfile("testit.bat", """\
+c:\\cputrapf
+rem end
+""", newline="\r\n")
+
+    # compile sources
+    mkcom("cputrapf", r"""
+.text
+.code16
+
+    .globl  _start16
+_start16:
+
+    push    %cs
+    pop     %ds
+
+    movw    $0x2501, %ax
+    movw    $int1, %dx
+    int     $0x21
+    pushw   $0x3003
+    pushw   $0x3303
+    popf
+    popf
+    nop
+
+    movb    $0x9, %ah              # print string
+    movw    $result, %dx
+    int     $0x21
+
+exit:
+    movw    $0x4c00, %ax
+    int     $0x21
+    ret
```

Comment:
Why not just ret?
Or do you mind the retcode?

Reply:
I don't really care about the retcode whilst in the suite, but if run standalone someone might.